testing/e2e/describes-e2e.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*eslint-disable */
+// import to install chromedriver
+import * as chromedriver from 'chromedriver';
+/*eslint-enable */
```

Comment:
Instead of tis, you can disable one or more `eslint` rules for a given line via `eslint-disable-line`. See [here](https://github.com/ampproject/amphtml/search?utf8=%E2%9C%93&q=eslint-disable-line) for examples.